CSharpOOP4/Shape.cs

```diff
@@ -0,0 +1,29 @@
+namespace CSharpOOP4;
+
+public abstract class Shape
+{
+    private string _name;
+
+    public string Name
+    {
+        get { return _name; }
+        set { _name = value; }
+    }
+
+    public Shape(string _name)
+    {
+        this._name = _name;
+    }
+
+    public Shape()
+    {
+        
+    }
+
+    public virtual void PrintShapeInfo()
+    {
+        Console.WriteLine($"The shape's name is {_name} and it's volume is {calculateVolume()}");
+    }
+
+    public abstract double calculateVolume();
```

Comment:
Name should be in uppercase

Reply:
Haven's seen any chabges. Change name to CalculateVolume

Reply:
The comment was unclear. Needed to be clarified like this: "name" of method. Because I also has a variable called "name" and it's not clear to which instance you refer to.